prac_05/hex_colours.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+Hex Colours Dictionary Lookup
+"""
+
+colour_codes = {
+        'blue': '#0018a8',
+        'cyan': '#00ffff',
+        'crimson': '#dc143c',
+        'daffodil': '#ffff31',
+        'ebony': '555d50',
+        'emerald': '#50c878',
+        'firebrick': '#ff3030',
+        'gray': '#bebebe',
+        'heliotrope': '#df73ff',
+        'magenta': '#ff00ff'
+        }
+
+while True:
```

Comment:
Another while True loop... have we ever used this kind of loop?

Look at the error checking loop pattern, should look something like while colour != ""